eth/fetcher/tx_fetcher.go

```diff
@@ -876,6 +885,12 @@ func (f *TxFetcher) scheduleFetches(timer *mclock.Timer, timeout chan struct{},
 				if len(hashes) >= maxTxRetrievals {
 					return false // break in the for-each
 				}
+				if meta != nil { // Only set eth/68 and upwards
+					bytes += uint64(meta.size)
+					if bytes >= maxTxRetrievalSize {
+						return false
+					}
+				}
```

Comment:
This will continue until it overflows. I guess that is the intention?

Reply:
Yes, since even a single blob tx can be way above the limit, we need to permit going over it.